modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+//
+// * installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc
+
+:_mod-docs-content-type: Procedure
+[id="installation-osp-local-disk-deployment_{context}"]
+= Deploying {rh-openstack} on local disk
+
+.Prerequisites
+
+* You have an OpenStack cloud with a working Cinder, and it has at least 75 GB of available storage to accommodate 3 root volumes for the OpenShift control plane.
+
+* The OpenStack cloud must be deployed with Nova ephemeral storage that uses a local storage backend and not RBD.
+
+
+.Procedure
+
+[WARNING]
+====
+This procedure can testing etcd on local disk and should not be used on production cluster because in certain cases complete loss of the control plane can occur. For more information, see "Overview of backup and restore operation" under "Backup and restore".
+====
+
+. Create a Nova flavor for the control plane with 10 GB of ephemeral disk by running the following command:
++
+[source,terminal]
+----
+$ openstack flavor create --ram 16384 --disk 25 --ephemeral 10 --vcpus 4 m1.xlarge
+----
+
+. Deploy a cluster with root volumes for the control plane; for example:
++
+.Example YAML file
+[source,yaml]
+----
+# ...
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: ${CONTROL_PLANE_FLAVOR}
+      rootVolume:
+        size: 25
+        types:
+        - ${CINDER_TYPE}
+  replicas: 3
+# ...
+----
+
+. Deploy the cluster you created by running the following command:
++
+[source,terminal]
+----
+$ openshift-install create cluster --dir <installation_directory> <1>
+----
++
+<1> For `<installation_directory>`, specify the location of your customized `./install-config.yaml file` that you created in step 2.
```

Comment:
```suggestion
<1> For `<installation_directory>`, specify the location of your customized `./install-config.yaml file` that you created previously.
```

Reply:
or "that was previously created" if we follow @ShaunaDiaz's suggestion on "you".